src/main/java/me/tongfei/progressbar/ProgressBarBuilder.java

```diff
@@ -52,7 +53,12 @@ public ProgressBarBuilder setUnit(String unitName, long unitSize) {
     }
 
     public ProgressBarBuilder showSpeed() {
+        return showSpeed(null);
+    }
+
+    public ProgressBarBuilder showSpeed(String speedFormat) {
```

Comment:
Setter for `speedFormat` has been added to `ProgressBarBuilder` already.

Reply:
One can pass the `speedFormat` to `showSpeed`. call `showSpeed` without argument as before is OK and the default speed format will be took.

Reply:
Got it. Thanks!